src/universal/components/CreateCard/CreateCard.js

```diff
@@ -1,11 +1,9 @@
 import React, {PropTypes} from 'react';
 import look, {StyleSheet} from 'react-look';
-// import FontAwesome from 'react-fontawesome';
 import PushButton from '../PushButton/PushButton';
 import Ellipsis from '../Ellipsis/Ellipsis';
 import Type from '../Type/Type';
 import theme from 'universal/styles/theme';
-import withHotkey from 'react-hotkey-hoc';
```

Comment:
i put this in the parent component, otherwise I'd need logic that checked to see if the handlers were passed through.